### ISSUE TITLE
@Delegate and @Builder collide when both classes are annotated with @Builder

### ISSUE THREAD

**Issue by timothyBrake** [projectlombok/lombok#1951]
Annotating both classes with `@Builder` gives exception when `@Delegate` is used
```
@Data
@Builder(toBuilder = true)
public class Foo {
    @Delegate
    private Bar bar;
}

@Data
@Builder(toBuilder = true)
public class Bar {
    private String name;
}
```
Gives exception:
`Error:(14, 5) java: method toBuilder() is already defined in class Foo`

An excludeMethodNames on `@Delegate` would be great

```
@Data
@Builder(toBuilder = true)
public class Foo {
    @Delegate(excludeMethods={"build","builder"}
    private Bar bar;
}

```

**Comment by andreasstrack** [comment]
Hi Timothy, 

I was actually able to work around this by using

`@Delegate(excludes = Builder.class)`

Hope this helps!

**Comment by Rawi01** [comment]
This was fixed in #2467
